addons/im_livechat/static/src/embed/chat_window/chat_window_container_patch.js

```diff
@@ -0,0 +1,15 @@
+/* @odoo-module */
+
+import { ChatWindowContainer } from "@mail/core/common/chat_window_container";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ChatWindowContainer.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.messaging.isReady.resolve({
+            channels: [],
+            current_user_settings: {},
+        });
+    },
```

Comment:
It would be better to do this in a service instead of a component, maybe `livechat_service`?